packages/algob/src/internal/core/tasks/builtin-tasks.ts

```diff
@@ -13,3 +13,10 @@ export default function (): void{
     loadPluginFile(path.join(basedir, t[1]));
   }
 }
+
+// checks if the task name is not a setup kind of task
+export function isSetupTask (taskName: string): boolean {
+  return taskName === tasks.TASK_HELP ||
```

Comment:
You check for HELP task three times.
Once it's during the start of cli method, then in the middle and now it's here. Can you check for it once?

Reply:
No, I can't check it once because the conditions are different. Here we are also checking for init. The method is not called `isHelpTask`.